### DISCUSSION TITLE
"Left button" and "Right button" can have multiple keyboard keys bound to them

### DISCUSSION THREAD

**Discussion by anexistantperson** [ppy/osu#14597] [Ideas] [answered]
**Describe the bug:** 
Due to Mouse inputs being unbindable from "Left button" and "Right button", You can somewhat bind more buttons to them. Causing a method of legitimately-cheating in comparison to stable. Although the buttons tend to override each other when doing this, they can still be used to stream at a normally-unrealistic speed.

**Screenshots or videos showing encountered issue:** 
![image](https://user-images.githubusercontent.com/55331139/131603112-8b74be99-aba9-44ed-ae8f-758e65092b17.png)

**osu!lazer version:** 
2021.828.0

**Comment by peppy** [comment]
This is intended behaviour. I'll move this to a discussion if you'd like to discuss further there.

**Comment by anexistantperson** [comment]
Oh. Are you trying to balance playstyles? I guess this is technically possible in standard if you actually use the mouse to click alongside the keyboard. But I still feel you're misunderstanding me, this does change normal keyboard + tablet/mouse gameplay.

**Comment by smoogipoo** [comment, marked as answer]
We've historically allowed players to bind more keys via external programs as long as the extra keys don't affect the already-pressed keys, such as releasing them. That is, if you bind Z and C to the same key, pressing C does _not_ release Z. This is already the way lazer works.

I don't think having more keys makes things easier, or at least not in all cases, as you'll have to juggle finger control vs raw speed with two pretty completely different play styles. Some mechanics will be difficult for 3-finger players, such as high BPM double/triple alternate bursts, quad cutstreams, or stacked circles under streams longer than 3 circles.
